bucket/defraggler.json

```diff
@@ -28,11 +36,17 @@
             ]
         }
     },
+    "pre_install": [
+        "if(!(Test-Path \"$persist_dir\\Defraggler.ini\")) {",
+        "   Set-Content \"$dir\\Defraggler.ini\" (@('[Software\\Piriform\\Defraggler]', 'UpdateCheck=0') -join \"`r`n\") -Encoding ASCII",
+        "}"
+    ],
+    "persist": "Defraggler.ini",
     "checkver": {
-        "url": "https://www.piriform.com/defraggler/download",
-        "re": "<strong>\\s*v([\\d.]+)</strong>"
+        "url": "https://www.ccleaner.com/defraggler/download",
+        "regex": "Latest version: <b>([\\d.]+)</b>"
     },
     "autoupdate": {
-        "url": "https://www.piriform.com/defraggler/download/portable/downloadfile#/dl.7z"
+        "url": "https://www.ccleaner.com/defraggler/download/portable/downloadfile#/dfsetup$majorVersion$minorVersion.zip"
```

Comment:
Why do you rename it with version in name?

```suggestion
        "url": "https://www.ccleaner.com/defraggler/download/portable/downloadfile#/dl.zip"
```

Reply:
![headers](https://user-images.githubusercontent.com/5832170/60560232-9359f900-9d81-11e9-995c-a3bafde6b29e.png)

`dl.zip` is okay, too.

Reply:
Manual renames using version number is irrelevant and it will not help anything, all manifests are using `dl` or name of manifest. Follow it.

change it in recuva, defragfler, speccy